website/docs/concepts2/mutations.mdx

```diff
@@ -37,12 +37,34 @@ final addTodo = Mutation<Todo>();
 Typically, this variable will either be global or as a `static final` variable on
 a [Notifier].
 :::
+You can additionally choose to call the mutation like a function,
```

Comment:
I just realised the page never spoje about this before.

It's probably worth moving it into its own section with a dedicated header.
And I'd put that under "Listening to mutations".